### PR TITLE
A11y talkback bug fixes

### PR DESCRIPTION
**Pull request by aishwarya56**
Talkback related bug fixes

### CODE REVIEW

File: FluentUI/src/main/java/com/microsoft/fluentui/persona/PersonaChipView.kt [hunk 1]
```diff
@@ -193,6 +198,29 @@ class PersonaChipView : TemplateView {
...
 
+    override fun getAccessibilityClassName(): CharSequence {
+        return CheckBox::class.java.name
+    }
+
+    override fun onInitializeAccessibilityNodeInfo(info: AccessibilityNodeInfo?) {
+        super.onInitializeAccessibilityNodeInfo(info)
+        info?.isEnabled = isEnabled
+        info?.isCheckable = true
+        info?.isChecked = isSelected
```
Comment (mnkshukl) [inline FluentUI/src/main/java/com/microsoft/fluentui/persona/PersonaChipView.kt:209]: can avoid ? in each line 

info?.apply{
}
}

File: FluentUI/src/main/java/com/microsoft/fluentui/persona/PersonaChipView.kt [hunk 2]
```diff
@@ -193,6 +198,29 @@ class PersonaChipView : TemplateView {
...
+        info?.text = name
+    }
+
+    override fun onInitializeAccessibilityEvent(event: AccessibilityEvent?) {
+        super.onInitializeAccessibilityEvent(event)
+        event?.isEnabled = isEnabled
```
Comment (mnkshukl) [inline FluentUI/src/main/java/com/microsoft/fluentui/persona/PersonaChipView.kt:215]: same as above

File: FluentUI/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -193,10 +193,22 @@
     <!--Describes the state of a menu item that is not checked.-->
     <string name="popup_menu_accessibility_item_state_not_checked">not checked</string>
 
+    <!-- *** Calendar *** -->
+
+    <!--accessibility-->
```
Comment (mnkshukl) [inline FluentUI/src/main/res/values/strings.xml:198]: is description tag not required anywhere here for accurate translation ?

File: FluentUI/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -185,10 +185,12 @@
         <item name="android:maxHeight">@dimen/fluentui_progress_linear_height</item>
     </style>
     <style name="Widget.FluentUI.LinearProgress.Determinate" parent="Widget.FluentUI.LinearProgress">
+        <item name="android:contentDescription">@string/progress_linear_content_description_determinate</item>
```
Comment (mnkshukl) [inline FluentUI/src/main/res/values/styles.xml:188]: formatting ?

File: FluentUI/src/main/java/com/microsoft/fluentui/calendar/CalendarAdapter.kt [hunk 5]
```diff
@@ -153,6 +153,7 @@ internal class CalendarAdapter : RecyclerView.Adapter<CalendarAdapter.CalendarDa
 
     override fun onClick(v: View) {
         onDateSelectedListener.onDateSelected((v as CalendarDayView).date.getLocalDateToZonedDateTime)
+        v.announceForAccessibility(v.contentDescription.toString()+context.getString(R.string.calendar_adapter_accessibility_item_selected))
```
Comment (mnkshukl) [inline FluentUI/src/main/java/com/microsoft/fluentui/calendar/CalendarAdapter.kt:156]: nit: concat using $ operator/ concat / string builder ?

File: FluentUI/src/main/res/values/strings.xml (hunk 3 above)
Comment (aishwarya56) [reply FluentUI/src/main/res/values/strings.xml:198]: Not used anywhere, but we can start using this or the. comment tag. What do. you suggest?

**Review by mnkshukl — APPROVED** [review]

**Pull request merged**